apps/admin/src/pages/index.tsx

```diff
@@ -7,22 +7,29 @@ import { WithNav } from "~/components/with-nav.jsx";
 import { HackathonAccountRegistration } from "./hackathon-account-registration.tsx";
 import { HackathonRegistration } from "./hackathon-registration";
 import { HackathonSignIn } from "./hackathon-sign-in";
+import { Hackathons } from "./hackathons";
 import { Hello } from "./hello";
 import { Overview } from "./overview";
 import { SignIn } from "./sign-in";
 import { SignUp } from "./sign-up";
 import { Users } from "./users";
+import { Sponsors } from "./sponsors";
 
 export function Pages() {
   return (
     <>
       <Switch>
         <AdminRoute path="/" component={WithNav(Overview)} />
         <AdminRoute path="/users" component={WithNav(Users)} />
+        <AdminRoute path="/sponsors" component={WithNav(Sponsors)} />
         <Route path="/hello" component={WithNav(Hello)} />
         <Route path="/sign-in" component={SignIn} />
         <Route path="/sign-up" component={SignUp} />
         <Route path="/hackathon/sign-in" component={HackathonSignIn} />
+        <AuthenticatedRoute
```

Comment:
This should use the `AdminRoute` component instead.

Reply:
Yes sir